src/targets/github.ts

```diff
@@ -187,57 +130,26 @@ export class GithubTarget extends BaseTarget {
     const isPreview =
       this.githubConfig.previewReleases && isPreviewRelease(version);
 
-    try {
-      const response = await this.github.repos.getReleaseByTag({
-        owner: this.githubConfig.owner,
-        repo: this.githubConfig.repo,
-        tag,
-      });
-      this.logger.warn(`Found the existing release for tag "${tag}"`);
-      return response.data;
-    } catch (e: any) {
-      if (e.status !== 404) {
-        throw e;
-      }
-      this.logger.debug(`Release for tag "${tag}" not found.`);
+    if (isDryRun()) {
+      this.logger.info(`[dry-run] Not creating the release`);
```

Comment:
Since this is a draft, I think it might be actually good to create the draft even in dry mode. It doesn't do anything destructive.

Reply:
I don't expect a dry run to have any side effects whatsoever, it should just log. 🐭